CONTRIBUTING.md

```diff
@@ -400,6 +402,28 @@ Once the app has started you should be able to pull up these URLs in your web br
 
 </details>
 
+# Adding a New Feature
+
+In order to understand where to start, it may help to familiarise yourself with our tech stack.  The key technologies we use are: PostgreSQL, TypeORM, Apollo, Express, React and Next.js.  For more details:
```

Comment:
```suggestion
In order to understand where to start, it may help to familiarize yourself with our [tech stack](https://github.com/freeCodeCamp/chapter/blob/main/README.md#tech-stack). For more details:
```
How about this?  Does it seem reasonable to have this link and a link back from the README?

Reply:
Sure, if you want to keep the bullet list for the Tech Stack in the README, then linking to/from each would be good.